lib/amorail/entities/elementable.rb

```diff
@@ -0,0 +1,36 @@
+module Amorail
+  module Elementable
+    extend ActiveSupport::Concern
+
+    ELEMENT_TYPES = [
```

Comment:
To be honest I just copy-pasted existent solution in constant.
If the structure doesn't matter I totally agree with you.